frontend/src/common/enums/app/app-route.enum.ts

```diff
@@ -6,12 +6,9 @@ enum AppRoute {
   EDUCATION = '/education',
   INTERVIEW = '/interviews',
   INTERVIEWS_$ID = '/interviews/:id',
-  SETTINGS_BILLING = '/settings/billing',
   SETTINGS_PROFILE = '/settings/profile',
```

Comment:
Then it should be just `profile`? Or it should be left as it is?

Reply:
agreed